bookshelf/api_v1.py

```diff
@@ -151,6 +151,18 @@ def arch():
     return result
 
 
+def disable_openssh_rdns(distribution):
+    log_green('Disabling openssh reverse dns lookups')
+    openssh_config_file = '/etc/ssh/sshd_config'
+    dns_config = 'UseDNS no'
+    if not file_contains(openssh_config_file, dns_config, use_sudo=True):
+        file_append(openssh_config_file, dns_config, use_sudo=True)
+        service_name = 'sshd'
+        if 'ubuntu' in distribution:
+            service_name = 'ssh'
```

Comment:
:question: Maybe make `service_name` a parameter and have separate code for figuring out the ssh service name based on distro. Take a look at https://github.com/ClusterHQ/flocker/blob/master/flocker/provision/_install.py if you want to see where this path leads.